modules/openxr/openxr_api.cpp

```diff
@@ -316,6 +317,58 @@ String OpenXRAPI::get_swapchain_format_name(int64_t p_swapchain_format) const {
 	return String("Swapchain format ") + String::num_int64(int64_t(p_swapchain_format));
 }
 
+void OpenXRAPI::set_object_name(XrObjectType p_object_type, uint64_t p_object_handle, const String &p_object_name) {
+	OpenXRDebugUtilsExtension *debug_utils = OpenXRDebugUtilsExtension::get_singleton();
+	if (!debug_utils) {
+		// Not enabled? Ignore.
+		return;
+	} else if (!debug_utils->get_active()) {
+		// Not active? Ignore.
+		return;
+	}
```

Comment:
Any reasons not to merge those two checks?

Reply:
Not really, probably makes sense to do them together